src/current/v22.2/install-cockroachdb-linux.md

```diff
@@ -152,9 +157,18 @@ true
 
 <div id="build-from-source-linux" class="install-option">
   <h2 id="install-source">Build from Source</h2>
-  <p>See the <a href="https://wiki.crdb.io/wiki/spaces/CRDB/pages/181338446/Getting+and+building+CockroachDB+from+source">public wiki</a> for guidance.</p>
+  <p>See the <a href="https://wiki.crdb.io/wiki/spaces/CRDB/pages/181338446/Getting+and+building+CockroachDB+from+source">public wiki</a> for guidance. When building on the ARM architecture, refer to <a href="#limitations">Limitations</a>.</p>
 </div>
 
+<h2 id="limitations">Limitations</h2>
+
+CockroachDB runtimes built for the ARM architecture have the following limitations:
+
+- CockroachDB on ARM is in [Limited Access](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases) in CockroachDB v22.2.13, and is **experimental** in all other versions. Experimental binaries and Docker images are not qualified for production use.
+- Clusters with a mix of Intel and ARM nodes are untested. Cockroach Labs recommends that all cluster nodes have identical CockroachDB versions, hardware, and software.
+- Floating point operations may yield different results on ARM than on Intel, particularly [Fused Multiply Add (FMA) intrinsics](https://developer.arm.com/documentation/dui0375/g/Compiler-specific-Features/Fused-Multiply-Add--FMA--intrinsics).
+- When [building from source](#install-source) on ARM, consider disabling FMA intrinsics in your compiler. For GCC, refer to [Options That Control Optimization](https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html) in the GCC documentation.
```

Comment:
I don't think building go sources with `GCC` is an effective option. As it currently stands, it's not possible to disable FMA intrinsics owing to the Go compiler [1]. Maybe, we just link that issue?

[1] https://github.com/golang/go/issues/36971#issuecomment-581874484